public/picker.js

```diff
@@ -1,6 +1,11 @@
 var categoryPicker = document.getElementById('categoryPicker');
 var sourcePicker = document.getElementById('sourcePicker');
 
+var supportedLngs = Object.keys(sources);
+var language = (supportedLngs.indexOf(i18next.language) > -1)
+    ? i18next.language
+    : supportedLngs[0];
```

Comment:
@dfarache shouldn't the i18n library decide this?

Reply:
I don't know... It'd be possible to have localisation support for several languages using i18n but still be using media sources from just 2 countries. As a matter of fact, the i18n and the sources are defined in different files in this project

Reply:
@dfarache I find this a bit hacky but I understand that we probably need this to get it fix quickly.

Could you just add a comment or create a ticket to make a proper implementation of this?